requirements.txt

```diff
@@ -2,4 +2,4 @@ pytest>=4.6
 pytest-cov>=2.6.1
 pytest-xdist>=1.25.0
 coveralls>=1.5.1
-
+antlr4-python3-runtime>=4.9.1
```

Comment:
Can we somehow make this optional?

Reply:
If not using .f support, it should be optional. Need to figure out how to mark it as optional.

Reply:
Doesn't seem like you can

https://stackoverflow.com/questions/3664478/optional-dependencies-in-a-pip-requirements-file

Reply:
We now have 2 requirements files, with an explanation of which one to use in `README.md`